test/integration/targets/virt_net/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+- include_vars: '{{ item }}'
+  with_first_found:
+    - "{{ ansible_distribution }}-{{ ansible_distribution_version}}.yml"
+    - "{{ ansible_distribution }}-{{ ansible_distribution_major_version}}.yml"
+    - "{{ ansible_distribution }}.yml"
+    - "default.yml"
+
+- block:
+    - include_tasks: 'prepare.yml'
```

Comment:
Why are these tasks in a separate file?

Reply:
It was to isolate the installation from the tests but Yeah, it's just two tasks. I merge the two files.